tdrs-frontend/src/configureStore.js

```diff
@@ -13,9 +14,29 @@ export const history = createBrowserHistory()
 export default function configureStore(preloadedState) {
   const middlewares = [thunkMiddleware, loggerMiddleware]
   const composedEnhancers = composeWithDevTools(applyMiddleware(...middlewares))
+  const devState = {
+    router: { location: { pathname: '/profile' } },
```

Comment:
I believe a few additional changes will be needed for the DEVAUTH feature. Hoping to get some time with @jtimpe today to figure this out.

Reply:
@tptignor does it make sense to move the devauth stuff to a separate PR if there is still going to be active work on it? That way this PR isn't blocked and the elastic indexes/documents can be created in the deployed envs?

Reply:
Hi guys. I actually put a ton of work in the last 12 hours into rewriting and verifying devauth. What's here now should be good to go, and it passes the unit tests that were passing before. However, if someone wants to recommend a different base branch to port it to, I completely understand and will get to that as soon as I can. (Need to get back to DIDDMS work now...)

Reply:
@tptignor @elipe17 i'd like to see these PRs split up just on the basis of having clean diffs and commit histories when we go to merge them. Both sets of changes currently look good to me though

Reply:
Pulled Dev Auth out of this PR.